packages/docs/src/root.tsx

```diff
@@ -48,13 +48,31 @@ export default component$(() => {
 
   useContextProvider(GlobalStore, store);
 
+  useVisibleTask$(
```

Comment:
Let's use `useOnDocument('DOMContentLoaded'` with `sync$()`
Thanks @thejackshelton for the suggestion

Reply:
I set it as an inlined script instead, this way we're sure it'll be executed before any other script. Shouldn't matter too much ^^.